docs/guides/BUILDING.md

```diff
@@ -161,7 +167,7 @@ RUN set -x \
 ENV ZAP_DEVELOPMENT_PATH=/opt/zap-${ZAP_VERSION}
 ```
 
-### Which ZAP to use
+### ZAP environment variables
```

Comment:
This isn't right.  The important part is not environment variables; the important part is that this is how which ZAP is used is determined.  The env vars are the mechanism, not the goal.